rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -2282,14 +2274,6 @@ namespace rsx
 		registers[0xa34 / 4] = 0x45000000;
 		registers[0xa38 / 4] = 0x3f000000;
 		registers[0xa3c / 4] = 0x0;
-		registers[NV4097_SET_VIEWPORT_OFFSET] = 0x45000000;
```

Comment:
idk why you remove it, but atleast comment it out because it has documentation purposes.

Reply:
It repeats twice on realhw due to hw bugs. I'll leave a comment.

Reply:
It repeats twice on realhw due to hw bugs. I'll leave a comment.